src/authentication/Login/Login.tsx

```diff
@@ -0,0 +1,26 @@
+import Logo from '../../public/am-logo.jpg';
+
```

Comment:
I'd recommend an SVG logo. I already shared it on slack; https://slack-files.com/T0YAREQ77-F04PJ0RQJSF-7525bc3f62

Reply:
Resolved